.github/pull_request_template.md

```diff
@@ -0,0 +1 @@
+Please make the PR title informative to the user experience. For example when adding a feature, say "Add metrics tab for Deployments" instead of "Bump Astro-UI to version X.Y.Z". Bugs are sometimes harder to make customer-facing because they are very detailed, but do what you can. Instead of "Fix metrics cardinality bug" (user does not know what 'cardinality' means and does not matter to them), say "Fix bug slowing down the metrics tab". The PR title will be used for the commit message when it's merged, and these commit messages are used to generate customer-facing release notes.
```

Comment:
@sjmiller609 we had some parallel thinking.  I don't know if you wanted to be more granular with this and divide some of this into sections.  I have an example in Houston-API https://github.com/astronomer/houston-api/blob/main/.github/pull_request_template.md I think it'd be nice if we have a similar format throughout our repos.